### ISSUE TITLE
Bug: sympify "weirdly" represents unevaluated terms begin with -, +1, or -1 

### ISSUE THREAD

**Issue by pstricks-fans** [sympy/sympy#24112]
I want to get unevaluated expression of `'-sqrt(7)/2 + 1/3 - 1/2'`. Unfortunately SymPy adds weird `-1` and `+1` multipliers that are not good for `sympy.latex()`.

```python
import sympy
a = sympy.sympify('-sqrt(7)/2 + 1/3 - 1/2',evaluate=False)
a
```

![image](https://user-images.githubusercontent.com/50270635/194123188-1cb46fab-5efe-4250-a40f-420ca12af7f7.png)

**Comment by oscarbenjamin** [comment]
It is possible to construct what you want:
```python
In [8]: init_printing(order='none')

In [9]: Add(-sqrt(7)/2, Rational(1, 3), Rational(-1, 2), evaluate=False)
Out[9]: 
  √7   1   1
- ── + ─ - ─
  2    3   2

In [10]: latex(_)
Out[10]: '- \\frac{\\sqrt{7}}{2} + \\frac{1}{3} - \\frac{1}{2}'
```
The problem is that parsers using evaluate=False will end up having spurious 1s. That should be fixed in the parsing code somehow.

**Comment by smichr** [comment]
I added cf to OP

**Comment by oscarbenjamin** [comment]
> I added cf to OP

I don't think that #24022 is relevant here. The issue is that the parsing code when used with evaluate=False introduces spurious `1 *` or `-1 *` that don't need to be there e.g; `1/3` should be parsed as `Rational(1, 3)` rather than as `Mul(Integer(1), Rational(1, 3))`.
